scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,53 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+---- Test results
+-- SELECT * FROM animal_release ar
+-- WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+-- 2. When capture_location is null, assign value of release_location
+UPDATE animal_release
+SET capture_location = release_location
+WHERE capture_location IS NULL;
```

Comment:
> just filling in the value displayed to the end user from either the capture_location or release_location field depending on which one is not-null

When users fill in the capture_location field in the gui, then the release_location field gets populated automatically with the same set of lat/long coordinates. Both fields are now mandatory in the production web app so we won't have this issue anymore.

> how can we know that it's not null because the person recording didn't have that information rather than it always being the case that capture and release locations coincide when one of the fields is null?

We know it's not null because we've got information for the capture and release *locality* fields (see https://github.com/aodn/aatams/issues/268#issuecomment-197084968)